packages/aws-cdk-lib/aws-ecs/README.md

```diff
@@ -372,6 +372,23 @@ container.addPortMappings({
 });
 ```
 
+Sometimes it is useful to be able to configure port ranges for a container, e.g. to run applications such as game servers
+and real-time streaming which typically require multiple ports to be opened simultaneously. This feature is supported on
+both Linux and Windows operating systems for both the EC2 and AWS Fargate launch types. There is a maximum limit of 100
+port ranges per container, and you cannot specify overlapping port ranges.
+
+Docker recommends that you turn off the `docker-proxy` in the Docker daemon config file when you have a large number of ports.
+For more information, see [Issue #11185](https://github.com/moby/moby/issues/11185) on the GitHub website.
+
+```ts
+declare const container: ecs.ContainerDefinition;
+
+container.addPortMappings({
+    containerPort: ecs.CONTAINER_PORT_UNSET_VALUE,
```

Comment:
We can make this look less awkward by giving it a name that sounds less like "we screwed up" and more like "that's just the way it works" 😉 .

If we name it something like `CONTAINER_PORT_USE_RANGE` or something, it quite literally says "use this other property for the port range" and most people won't think twice about it.

One more thing: module level constants will not be exposed over jsii, so this needs to be a `public static readonly` value on some class somewhere. You can define a fresh one if you want:

```ts
export class Constants {
  public static readonly CONTAINER_PORT_USE_RANGE = 0;
}
```

Reply:
>this needs to be a `public static readonly` value on some class somewhere

I'd like to see such a constant in the `PortMapping` interface as it belongs to it, but since that's not possible, would it be okay to have it in the `ContainerDefinition` class?

Reply:
Yep, fine by me as well!